manifests/l/lhvy/pipes-rs/1.6.0/lhvy.pipes-rs.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: lhvy.pipes-rs
+PackageVersion: 1.6.0
+InstallerLocale: en-US
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: pipes-rs.exe
+  PortableCommandAlias: pipes-rs
+Installers:

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
The package installs, but requires the dependency to run:
![image](https://user-images.githubusercontent.com/114952565/218530676-eca8cbfe-449f-4968-8a1f-803258d26725.png)